ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/flow_tracking.py

```diff
@@ -24,13 +26,20 @@ def flow_tracking(self) -> dict | None:
         if not self.shared_utils.is_cv_pathfinder_router:
             return None
 
+        flow_tracking = get(self._hostvars, "cv_pathfinder_flow_tracking", default={})
+        exporter = get(flow_tracking, "exporter", default={})
+        exporter_name = get(exporter, "name", default="DPI-EXPORTER")
+        template_interval = get(exporter, "template_interval", default=3000000)
```

Comment:
```suggestion
        exporter_name = get(flow_tracking, "exporter.name", default="DPI-EXPORTER")
        template_interval = get(flow_tracking, "exporter.template_interval", default=3000000)
```

Reply:
3000000 -> we are changing the current default value, any reason?

Reply:
Because that's what you mentioned on 117.